src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13904,6 +13904,19 @@ namespace System.Runtime.InteropServices.Swift
         public unsafe SwiftSelf(void* value) { throw null; }
         public unsafe void* Value { get { throw null; } }
     }
+    public readonly partial struct SwiftSelf<T> where T: unmanaged
+    {
+        private readonly int _dummyPrimitive;
```

Comment:
```suggestion
        private readonly T _dummyPrimitive;
```

Using `T` here allows the C# compiler to detect infinite generic expansion at build time.